test/fixtures/typescript/math_spec.ts

```diff
@@ -5,6 +5,10 @@ const { add } = math
 
 const x: number = 3
 
+const isKeyOf = <T>(obj: T, key: any): key is keyof T => {
```

Comment:
Let's add a comment with a link to the issue above this to make it clear what this is testing:

```
// ensures that generics can be properly compiled and not treated
// as react components
// https://github.com/cypress-io/cypress-browserify-preprocessor/issues/44
```

Reply:
Done.